documentation/README.md

```diff
@@ -0,0 +1,80 @@
+
+If the recipes are simple, they should be included in the docstrings of the relevant functions/classes (and they [should be tested as part of the CI](https://docs.pytest.org/en/7.1.x/how-to/doctest.html)). When the recipes are more ellaborated (e.g., they require some setup that needs explanation, other packages, or multiple modules), we can create a *Cookbook* section in the documentation and add them as notebooks.
+
+### API Reference
+
+- All public functions and methods should be documented. We use the [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) format
+- At least one example (that users can copy paste) should be provided
+
+## Tools
+
+We're migrating all our documentation websites to [Jupyterbook](https://jupyterbook.org/en/stable/intro.html); however, some still use Sphinx.
+
+## General Guidelines
+
+### Style
+
+Writing documentation is hard, so ensure you proofread several times and use tools like Grammarly to simplify writing.
+
+### Naming
+
+The names of the sections should be about what the user knows, not about the library's internals. For example, sklearn-evaluation has a `plot` module to create Machine Learning model evaluation plots. However, the practitioner might not know what plot they need; but they know what to achieve. So we should name our documentation sections accordingly (e.g., name a User Guide "Evaluating Feature Importances" instead of the plot name).
+
+### Hyperlinks
+
+Sections should be hyperlinked. For example, the API reference should link to the relevant Getting Started and User Guides. Furthermore, the Getting Started and User Guide should connect to specific parts of the API reference. For example, if the User Guide describes the use of the `plot.confusion_matrix` function, it should link to its documentation.
+
+### Notebooks (applies to Getting Started and User Guide)
+
+- Notebooks should not hide any important details (e.g., hiding cells) since users are expected to copy and paste the code on them
+- In some cases (e.g., when output is too verbose), we might decide to hide some outputs, but we should be cautious of these
+- When notebooks require optional dependencies, we should document them at the top of the file so that users can install the appropriate libraries (e.g., a top cell with: `%pip install somepackage --quiet`)
+- The notebook should be organized into short sections so that users can navigate to any specific one. The titles should be short but descriptive enough
+- Since we'll run the notebooks as part of the documentation compilation process, all of them should run fast (ideally, below 10 seconds)
+- If notebooks require external files we should include both a link to the file and a command to download it (e.g., `curl -O https://ploomber.io/some-file.txt`)
```

Comment:
Missing a point for getting started: notebooks should be short, easy to understand and without errors - this should be tested in the CI.

Reply:
short and clear: this is mentioned in the top section
will mention the CI part